doc/cephfs/fs-nfs-exports.rst

```diff
@@ -324,7 +327,7 @@ previous state of the export where possible.
 
 ::
 
-   $ ceph nfs export apply -i update_cephfs_export.json
+   $ ceph nfs export apply mynfs -i update_cephfs_export.json
```

Comment:
Please update the `ceph nfs export apply` command above to include `cluster_id`

Reply:
and please update doc about pool name change here
https://docs.ceph.com/en/latest/cephfs/fs-nfs-exports/#ganesha-configuration-hierarchy
https://docs.ceph.com/en/latest/cephfs/fs-nfs-exports/#set-customized-nfs-ganesha-configuration (see the note)

Reply:
fixed!